docs/installation.md

```diff
@@ -0,0 +1,210 @@
+# Installing JAX
+
+JAX is written in pure Python, but it depends on XLA, which needs to be
+installed as the `jaxlib` package. Use the following instructions to install a
+binary package with `pip` or `conda`, to use a
+[Docker container](#docker-containers-nvidia-gpu), or to [build JAX from
+source](developer.md#building-from-source).
+
+
+We support installing or building `jaxlib` on Linux (Ubuntu 20.04 or later) and
+macOS (10.12 or later) platforms. There is also *experimental* native Windows
+support.
+
+Windows users can use JAX on CPU and GPU via the [Windows Subsystem for
+Linux](https://docs.microsoft.com/en-us/windows/wsl/about), or alternatively
+they can use the *experimental* native Windows CPU-only support.
+
+### pip installation: CPU
```

Comment:
Second-level headings will show up in the sidebar: perhaps we should have short and meaningful headings like CPU, GPU, TPU, etc. to improve navigability